uber/site_sections/registration.py

```diff
@@ -623,7 +623,7 @@ def update_nonshift(self, session, id, nonshift_hours):
         if not re.match('^[0-9]+$', nonshift_hours):
             raise HTTPRedirect('shifts?id={}&message={}', attendee.id, 'Invalid integer')
         else:
```

Comment:
Since there are some computations performed during ``presave_adjustments``, we actually need this to be an integer so we can e.g. add it to other integers.